### ISSUE TITLE
New Class did not appear until page refresh

### ISSUE THREAD

**Issue by callmeyagr** [RotorHazard/RotorHazard#934]
**Confirm the issue is with RotorHazard and not a plugin**
Yes, not a plugin issue. Same on Google Chrome(127.0.6533.89) and Firefox(128.0.3) on NuclearHazard or standalone application installes on PC.

**Describe the bug**
Format -> Classes and Heats -> Add Class - new class did not appear until page refreshed, or duplicate class that already exist.

**To Reproduce**
1.Format -> Classes and Heats -> Add Class
2.Got nothing, until page refresh or press Duplicate(two classes appears - empty one and duplicated)

**Expected behavior**
New class will be created and you can add Hears with pilots

**Log Dump**
[rh_logs_20240805_125442.zip](https://github.com/user-attachments/files/16494119/rh_logs_20240805_125442.zip)

https://github.com/user-attachments/assets/85432226-66ca-47e1-a07f-d7c9f3063658

**Desktop (please complete the following information):**
WIN10 x64.

**Comment by HazardCreative** [comment]
I'm not able to reproduce the issue, even with a very large database open. Are there any errors in the browser console log (Ctrl+Shift+J) ?

**Comment by YegorYegor** [comment]
Yes, in fact new class appears if i wait 10-15 seconds. And as soon as it happen,  i got errors on console - Duplicate form field id in the same form.
![photo_2024-12-20_13-45-58](https://github.com/user-attachments/assets/bcd0fb39-0d71-4b84-a5c7-bb87a6c023f1)

**Comment by callmeyagr** [comment]
Works fine on 4.3.0-rc4.
